src/array_api_stubs/_draft/info.py

```diff
@@ -0,0 +1,163 @@
+__all__ = [
+    "__array_namespace_info__",
+    "capabilities",
+    "default_device",
+    "default_dtypes",
+    "devices",
+    "dtypes",
+]
+
+from ._types import (
+    Optional,
+    Union,
+    Tuple,
+    List,
+    device,
```

Comment:
I stared this `device` for a few seconds, and realized that elsewhere (e.g. `array_object.py`) we do `import device as Device` and use the capitalized name. Nothing is wrong, just a minor note 🙂

Reply:
Alas, in the creation function documentation, we use lowercase `device`, similar to `dtype`, and `array`. I cannot recall why we aliased as `Dtype` and `Device` in the array object spec. 😞

Reply:
I'll mark this as resolved, as the only place where we alias is in `array_object.py`, which is, I believe, to avoid name conflicts and make Sphinx happy.